### PR TITLE
unify collapsiples toggle button

### LINKED ISSUE #4122: Table collapsibles behavior improvement
Collapsible rows should be alligned throughout the app. Atm there are multiple implementations. Open for suggestions.

### PR DESCRIPTION
**Pull request by matborowczyk**
# Description

* Short description here *

closes #4122 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
- [x] Code is clear and sufficiently documented
- [x] Sufficient test cases (reproduces the bug/tests the requested feature)
- [x] Correct, in line with design
~~- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )~~

# Reviewer Checklist:

- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Code is clear and sufficiently documented
- [ ] Correct, in line with design
- [ ] 
Screenshot 2022-11-04 at 15 52 15

### CODE REVIEW

File: src/Slices/Resource/UI/ResourcesPage/ResourcesTable.tsx [hunk 1]
```diff
@@ -65,7 +65,10 @@ export const ResourcesTable: React.FC<Props> = ({
   return (
     <TableComposable {...props} variant={TableVariant.compact}>
       <Thead>
-        <Tr>{heads}</Tr>
+        <Tr>
+          <Th></Th>
```
Comment (LauraStordeur) [inline src/Slices/Resource/UI/ResourcesPage/ResourcesTable.tsx:69]: why the empty tag?

Comment (matborowczyk) [reply src/Slices/Resource/UI/ResourcesPage/ResourcesTable.tsx:69]: without it toggle cell move everything out of it's column

Comment (LauraStordeur) [reply src/Slices/Resource/UI/ResourcesPage/ResourcesTable.tsx:69]: Is this like a known issue? Is it used in other places too?

Comment (matborowczyk) [reply src/Slices/Resource/UI/ResourcesPage/ResourcesTable.tsx:69]: I wouldn't say it is an issue, more like how table work, the same solution is used in service inventory, other implementation use custom Toggle button with additional spanon header

Comment (LauraStordeur) [reply src/Slices/Resource/UI/ResourcesPage/ResourcesTable.tsx:69]: ok!

**Review by LauraStordeur — APPROVED** [review]

**Pull request closed without merging**